src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterHelper.cs

```diff
@@ -199,35 +202,37 @@ public static void UpdateCoreTypeBlobs(string testDataFilePath, string[] blobs)
             File.WriteAllLines(testDataFilePath, updatedTestDataLines);
         }
 
-        public static byte[] SerializeObjectToRaw(object obj)
+        public static byte[] SerializeObjectToRaw(object obj, FormatterAssemblyStyle assemblyStyle = FormatterAssemblyStyle.Simple)
```

Comment:
Why default values? Is it worth being explicit in all the callers, so it's clear?